.env

```diff
@@ -1 +1,10 @@
-PUBLIC_URL='/'
+PUBLIC_URL="/"
+REACT_APP_URL="http://localhost:3000/"
+REACT_APP_API_URL="https://api-dev.pinster.io"
+REACT_APP_IMAGE_SERVICE_API_URL="https://images.image-service-dev.pinster.io"
+REACT_APP_AUTH0_KEY="SECRET"
+REACT_APP_AUTH0_CALLBACK="http://localhost:3000/callback"
+REACT_APP_AUTH0_DOMAIN="dev-pinster-illuminusltd.auth0.com"
+REACT_APP_AUTH0_AUDIENCE="https://api-dev.pinster.io"
+REACT_APP_AUTH0_SCOPE="openid profile"
+REACT_APP_AUTH0_RESPONSE_TYPE="token id_token"
```

Comment:
Hey can you add these to the `.circleci/config.yml`? There's an `environment:` section that these can go in